src/modules/Microsoft.Storage/storageAccounts/blobServices/containers/deploy.bicep

```diff
@@ -12,9 +12,6 @@ param blobServicesName string = 'default'
 @description('Required. The name of the storage container to deploy.')
 param name string
 
-@description('Optional. Name of the immutable policy.')
```

Comment:
This template was copied from an ARM Bicep registry. We don't own it, but I wanted to remove the aspects that we don't need because this will get shared in Azure Quickstart Templates when we publish our templates.